lib/puppet/functions/stdlib/ssl/extract_pkey.rb

```diff
@@ -0,0 +1,9 @@
+require 'openssl'
+# @summary
+#   Extract a public key from a certificate
+Puppet::Functions.create_function(:'stdlib::ssl::extract_pkey') do
```

Comment:
im not sure if good practrice to add addtional nam spacing like this i.e. `stdlib::ssl::...`